include/deal.II/grid/grid_tools.h

```diff
@@ -3557,8 +3568,6 @@ namespace GridTools
         }
 
       // make sure that all communication is finished
-      // when we leave this function.
-      if (requests.size() > 0)
```

Comment:
What happened here?

Reply:
I don't know. I haven't touched those lines. My guess is that I am working on an old version from two days ago.

Reply:
Just re-add these two lines, and then combine (squash or fix-up) all of your commits into one.

Reply:
Done So. However, my local repo was completely fine before syncing, so I assumed I got that from the main repository. If you have an explanation for how my local repository differs from the remote after I pushed my changes, I'd be glad.